cli/src/org/partiql/cli/main.kt

```diff
@@ -131,6 +151,9 @@ private val outputFormatOpt = optParser.acceptsAll(listOf("output-format", "of")
  * Options:
  * * -e --environment: takes an environment file to load as the initial global environment
  * * -p --permissive: run the query in permissive typing mode (returns MISSING rather than error for data type
+ * * -t --typed-op-behavior: indicates how CAST should behave: (default: LEGACY) [LEGACY, HONOR_PARAMETERS]
+ * * -r --projection-iter-behavior: Controls the behavior of ExprValue.iterator in the projection result: (default: FILTER_MISSING) [FILTER_MISSING, UNFILTERED]
+ * * -v --undefined-variable-behavior: Defines the behavior when a non-existent variable is referenced: (default: ERROR) [ERROR, MISSING]
```

Comment:
Could some tests also be added for these other compile options? Something similar to what was done in [CliTest.kt](https://github.com/partiql/partiql-lang-kotlin/blob/b3e62fc676740fd18ffcb93099e55a1af14f9136/cli/test/org/partiql/cli/CliTest.kt#L213-L224) for the permissive CLI mode.

Reply:
Sure. Will do.